<ui version="4.0" >
 <class>ModifyProgram</class>
 <widget class="QDialog" name="ModifyProgram" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Program Modification</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../resources/resources.qrc" >:/images/images/tv.png</iconset>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QGroupBox" name="programName" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="MinimumExpanding" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>Program</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <layout class="QVBoxLayout" >
        <item>
         <layout class="QHBoxLayout" >
          <item>
           <layout class="QVBoxLayout" >
            <item>
             <widget class="QLabel" name="label_2" >
              <property name="text" >
               <string>Start:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_3" >
              <property name="text" >
               <string>End</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" >
            <item>
             <widget class="QDateEdit" name="beginDate" >
              <property name="calendarPopup" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateEdit" name="endDate" >
              <property name="calendarPopup" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" >
            <item>
             <widget class="QTimeEdit" name="beginHour" >
              <property name="calendarPopup" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTimeEdit" name="endHour" >
              <property name="calendarPopup" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" >
            <item>
             <widget class="QCheckBox" name="checkBoxBefore" >
              <property name="text" >
               <string>start</string>
              </property>
              <property name="checked" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBoxAfter" >
              <property name="text" >
               <string>end</string>
              </property>
              <property name="checked" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" >
            <item>
             <widget class="QSpinBox" name="before" >
              <property name="value" >
               <number>0</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="after" >
              <property name="value" >
               <number>0</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" >
            <item>
             <widget class="QLabel" name="label_5" >
              <property name="text" >
               <string>minutes before</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_4" >
              <property name="text" >
               <string>minutes after</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" >
          <item row="0" column="0" >
           <widget class="QLabel" name="label_8" >
            <property name="text" >
             <string>Recording directory:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QLineEdit" name="directory" />
          </item>
          <item row="0" column="2" >
           <widget class="QToolButton" name="directoryButton" >
            <property name="text" >
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QLabel" name="label" >
            <property name="text" >
             <string>Filename :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="filename" />
          </item>
         </layout>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/resources.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ModifyProgram</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>283</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel" >
     <x>283</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ModifyProgram</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>283</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel" >
     <x>283</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
